core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -389,7 +389,7 @@ private[spark] class TaskSchedulerImpl(
       maxLocality: TaskLocality,
       shuffledOffers: Seq[WorkerOffer],
       availableCpus: Array[Int],
-      availableResources: Array[Map[String, Buffer[String]]],
+      availableResources: Array[ExecutorResourcesAmounts],
```

Comment:
did the caller of this get updated?  it doesn't look like it unless I'm missing it or my ide didn't update

Reply:
The resources in WorkerOffer https://github.com/apache/spark/pull/43494/files#diff-942455d6c015aaa273f66a9d8ea00297d7c05e45e83a0761783c68ad134b1e87R33 has been changed to ExecutorResourcesAmounts